src/scratchtocatrobat/scratch/scratch3visitor/test_scratch3.py

```diff
@@ -33,6 +42,20 @@ def addInputOfType(block, key, type):
 
     block.inputs[key] = value
 
+def addFieldOfType(block, key, type):
```

Comment:
Could you please refactor the type_xyz to two different enums, like field_type and block_type? Otherwise, someone in future might confuse those 2 and call your method with brick types and vice versa.